doc/source/run.rst

```diff
@@ -0,0 +1,38 @@
+Running Tenks
+=============
+
+Commands
+--------
+
+Tenks has a variable ``cmd`` which specifies the command to be run. This
```

Comment:
This is no longer required, right?

Reply:
Also wonder if it would make sense to set the default of cmd to be deploy, so that running individual playbooks works as expected?

Reply:
It is still required - it's just that it's automatically set in the deploy.yml and teardown.yml playbooks. If you want to run individual playbooks, you need to ensure `cmd` is set correctly. It already defaults to `deploy` in host_vars/localhost (which is overridden by the set_facts in deploy.yml and teardown.yml, which in turn are overridden by a value of `cmd` specified in overrides.yml).

Reply:
Ok, let's leave it.